[receiver/activedirectoryds] Broken test `TestScrape`

### Component(s)

receiver/activedirectoryds

### Describe the issue you're reporting

#34492 broke some Windows-only tests. See example run here: https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/10316705082/job/28559486157

```
=== Failed
=== FAIL: . TestScrape/Fully_successful_scrape (0.30s)
make[2]: *** [../../Makefile.Common:126: test] Error 1
    scraper_test.go:47: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver/scraper_test.go:47
        	Error:      	Received unexpected error:
        	            	the following errors occurred:
        	            	 -  resource "map[]": missing expected scope: otelcol/activedirectorydsreceiver
        	            	 -  resource "map[]": unexpected scope: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
        	Test:       	TestScrape/Fully_successful_scrape

=== FAIL: . TestScrape/Scrape_with_errors (0.32s)
    scraper_test.go:82: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver/scraper_test.go:82
        	Error:      	Received unexpected error:
make[1]: *** [Makefile:179: receiver/activedirectorydsreceiver] Error 2
        	            	the following errors occurred:
        	            	 -  resource "map[]": missing expected scope: otelcol/activedirectorydsreceiver
        	            	 -  resource "map[]": unexpected scope: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
make[1]: *** Waiting for unfinished jobs....
        	Test:       	TestScrape/Scrape_with_errors

=== FAIL: . TestScrape (0.00s)

=== FAIL: . TestScrape/Fully_successful_scrape (re-run 1) (0.04s)
    scraper_test.go:47: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver/scraper_test.go:47
        	Error:      	Received unexpected error:
        	            	the following errors occurred:
        	            	 -  resource "map[]": missing expected scope: otelcol/activedirectorydsreceiver
        	            	 -  resource "map[]": unexpected scope: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
        	Test:       	TestScrape/Fully_successful_scrape

=== FAIL: . TestScrape (re-run 1) (0.00s)

=== FAIL: . TestScrape/Scrape_with_errors (re-run 1) (0.10s)
    scraper_test.go:82: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver/scraper_test.go:82
        	Error:      	Received unexpected error:
        	            	the following errors occurred:
        	            	 -  resource "map[]": missing expected scope: otelcol/activedirectorydsreceiver
        	            	 -  resource "map[]": unexpected scope: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
        	Test:       	TestScrape/Scrape_with_errors

=== FAIL: . TestScrape (re-run 1) (0.00s)

```

--- a/receiver/activedirectorydsreceiver/scraper_test.go
+++ b/receiver/activedirectorydsreceiver/scraper_test.go
@@ -45,7 +45,7 @@ func TestScrape(t *testing.T) {
 		require.NoError(t, err)
 
 		require.NoError(t, pmetrictest.CompareMetrics(expectedMetrics, scrapeData, pmetrictest.IgnoreStartTimestamp(),
-			pmetrictest.IgnoreTimestamp()))
+			pmetrictest.IgnoreTimestamp(), pmetrictest.IgnoreMetricDataPointsOrder()))
 
 		err = scraper.shutdown(context.Background())
 		require.NoError(t, err)
@@ -80,7 +80,7 @@ func TestScrape(t *testing.T) {
 		require.NoError(t, err)
 
 		require.NoError(t, pmetrictest.CompareMetrics(expectedMetrics, scrapeData, pmetrictest.IgnoreStartTimestamp(),
-			pmetrictest.IgnoreTimestamp()))
+			pmetrictest.IgnoreTimestamp(), pmetrictest.IgnoreMetricDataPointsOrder()))
 
 		err = scraper.shutdown(context.Background())
 		require.NoError(t, err)

--- a/receiver/activedirectorydsreceiver/testdata/golden_scrape.yaml
+++ b/receiver/activedirectorydsreceiver/testdata/golden_scrape.yaml
@@ -12,22 +12,22 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: client
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 21
                   attributes:
                     - key: type
                       value:
                         stringValue: server
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{binds}/s'
           - description: The amount of time taken for the last successful LDAP bind.
             gauge:
               dataPoints:
                 - asInt: "29"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             name: active_directory.ds.ldap.bind.last_successful.time
             unit: ms
           - description: The number of successful LDAP binds per second.
@@ -36,8 +36,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asDouble: 30
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{binds}/s'
           - description: The number of connected LDAP client sessions.
             name: active_directory.ds.ldap.client.session.count
@@ -45,8 +45,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "28"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{sessions}'
           - description: The number of LDAP searches per second.
             name: active_directory.ds.ldap.search.rate
@@ -54,15 +54,15 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asDouble: 31
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{searches}/s'
           - description: The percentage of directory object name component lookups that are satisfied by the Directory System Agent's name cache.
             gauge:
               dataPoints:
                 - asDouble: 22
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             name: active_directory.ds.name_cache.hit_rate
             unit: '%'
           - description: The number of pending update notifications that have been queued to push to clients.
@@ -71,8 +71,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "23"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{notifications}'
           - description: The number of operations performed per second.
             name: active_directory.ds.operation.rate
@@ -84,22 +84,22 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: read
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
-                - asDouble: 18
-                  attributes:
-                    - key: type
-                      value:
-                        stringValue: write
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 19
                   attributes:
                     - key: type
                       value:
                         stringValue: search
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asDouble: 18
+                  attributes:
+                    - key: type
+                      value:
+                        stringValue: write
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{operations}/s'
           - description: The amount of network data transmitted by the Directory Replication Agent.
             name: active_directory.ds.replication.network.io
@@ -114,8 +114,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: compressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "1"
                   attributes:
                     - key: direction
@@ -124,8 +124,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: uncompressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "2"
                   attributes:
                     - key: direction
@@ -134,8 +134,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: compressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "3"
                   attributes:
                     - key: direction
@@ -144,8 +144,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: uncompressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
               isMonotonic: true
             unit: By
           - description: The number of objects transmitted by the Directory Replication Agent per second.
@@ -158,15 +158,15 @@ resourceMetrics:
                     - key: direction
                       value:
                         stringValue: received
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 6
                   attributes:
                     - key: direction
                       value:
                         stringValue: sent
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{objects}/s'
           - description: The number of pending replication operations for the Directory Replication Agent.
             name: active_directory.ds.replication.operation.pending
@@ -174,8 +174,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "13"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{operations}'
           - description: The number of properties transmitted by the Directory Replication Agent per second.
             name: active_directory.ds.replication.property.rate
@@ -187,15 +187,15 @@ resourceMetrics:
                     - key: direction
                       value:
                         stringValue: received
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 8
                   attributes:
                     - key: direction
                       value:
                         stringValue: sent
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{properties}/s'
           - description: The number of objects remaining until the full sync completes for the Directory Replication Agent.
             name: active_directory.ds.replication.sync.object.pending
@@ -203,35 +203,35 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "4"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{objects}'
           - description: The number of sync requests made by the Directory Replication Agent.
             name: active_directory.ds.replication.sync.request.count
             sum:
               aggregationTemporality: 2
               dataPoints:
-                - asInt: "14"
-                  attributes:
-                    - key: result
-                      value:
-                        stringValue: schema_mismatch
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
-                - asInt: "15"
-                  attributes:
-                    - key: result
-                      value:
-                        stringValue: success
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
                 - asInt: "-13"
                   attributes:
                     - key: result
                       value:
                         stringValue: other
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asInt: "14"
+                  attributes:
+                    - key: result
+                      value:
+                        stringValue: schema_mismatch
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asInt: "15"
+                  attributes:
+                    - key: result
+                      value:
+                        stringValue: success
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
               isMonotonic: true
             unit: '{requests}'
           - description: The number of values transmitted by the Directory Replication Agent per second.
@@ -247,8 +247,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: distingushed_names
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 1
                   attributes:
                     - key: direction
@@ -257,8 +257,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: other
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 11
                   attributes:
                     - key: direction
@@ -267,8 +267,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: distingushed_names
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 1
                   attributes:
                     - key: direction
@@ -277,8 +277,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: other
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{values}/s'
           - description: The number of security descriptor propagation events that are queued for processing.
             name: active_directory.ds.security_descriptor_propagations_event.queued
@@ -286,28 +286,28 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "24"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{events}'
           - description: The rate of sub-operations performed.
             name: active_directory.ds.suboperation.rate
             sum:
               aggregationTemporality: 2
               dataPoints:
-                - asDouble: 26
-                  attributes:
-                    - key: type
-                      value:
-                        stringValue: security_descriptor_propagations_event
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
                 - asDouble: 25
                   attributes:
                     - key: type
                       value:
                         stringValue: search
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asDouble: 26
+                  attributes:
+                    - key: type
+                      value:
+                        stringValue: security_descriptor_propagations_event
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{suboperations}/s'
           - description: The number of threads in use by the directory service.
             name: active_directory.ds.thread.count
@@ -315,9 +315,9 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "27"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{threads}'
         scope:
-          name: otelcol/activedirectorydsreceiver
+          name: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
           version: latest

--- a/receiver/activedirectorydsreceiver/testdata/partial_scrape.yaml
+++ b/receiver/activedirectorydsreceiver/testdata/partial_scrape.yaml
@@ -12,22 +12,22 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: client
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 21
                   attributes:
                     - key: type
                       value:
                         stringValue: server
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{binds}/s'
           - description: The amount of time taken for the last successful LDAP bind.
             gauge:
               dataPoints:
                 - asInt: "29"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             name: active_directory.ds.ldap.bind.last_successful.time
             unit: ms
           - description: The number of successful LDAP binds per second.
@@ -36,8 +36,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asDouble: 30
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{binds}/s'
           - description: The number of connected LDAP client sessions.
             name: active_directory.ds.ldap.client.session.count
@@ -45,8 +45,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "28"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{sessions}'
           - description: The number of LDAP searches per second.
             name: active_directory.ds.ldap.search.rate
@@ -54,15 +54,15 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asDouble: 31
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{searches}/s'
           - description: The percentage of directory object name component lookups that are satisfied by the Directory System Agent's name cache.
             gauge:
               dataPoints:
                 - asDouble: 22
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             name: active_directory.ds.name_cache.hit_rate
             unit: '%'
           - description: The number of pending update notifications that have been queued to push to clients.
@@ -71,8 +71,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "23"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{notifications}'
           - description: The number of operations performed per second.
             name: active_directory.ds.operation.rate
@@ -84,22 +84,22 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: read
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
-                - asDouble: 18
-                  attributes:
-                    - key: type
-                      value:
-                        stringValue: write
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 19
                   attributes:
                     - key: type
                       value:
                         stringValue: search
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asDouble: 18
+                  attributes:
+                    - key: type
+                      value:
+                        stringValue: write
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{operations}/s'
           - description: The amount of network data transmitted by the Directory Replication Agent.
             name: active_directory.ds.replication.network.io
@@ -114,8 +114,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: compressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "1"
                   attributes:
                     - key: direction
@@ -124,8 +124,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: uncompressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "2"
                   attributes:
                     - key: direction
@@ -134,8 +134,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: compressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asInt: "3"
                   attributes:
                     - key: direction
@@ -144,8 +144,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: uncompressed
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
               isMonotonic: true
             unit: By
           - description: The number of objects transmitted by the Directory Replication Agent per second.
@@ -158,15 +158,15 @@ resourceMetrics:
                     - key: direction
                       value:
                         stringValue: received
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 6
                   attributes:
                     - key: direction
                       value:
                         stringValue: sent
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{objects}/s'
           - description: The number of pending replication operations for the Directory Replication Agent.
             name: active_directory.ds.replication.operation.pending
@@ -174,8 +174,8 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "13"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{operations}'
           - description: The number of properties transmitted by the Directory Replication Agent per second.
             name: active_directory.ds.replication.property.rate
@@ -187,42 +187,42 @@ resourceMetrics:
                     - key: direction
                       value:
                         stringValue: received
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 8
                   attributes:
                     - key: direction
                       value:
                         stringValue: sent
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{properties}/s'
           - description: The number of sync requests made by the Directory Replication Agent.
             name: active_directory.ds.replication.sync.request.count
             sum:
               aggregationTemporality: 2
               dataPoints:
-                - asInt: "14"
-                  attributes:
-                    - key: result
-                      value:
-                        stringValue: schema_mismatch
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
-                - asInt: "15"
-                  attributes:
-                    - key: result
-                      value:
-                        stringValue: success
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
                 - asInt: "-13"
                   attributes:
                     - key: result
                       value:
                         stringValue: other
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asInt: "14"
+                  attributes:
+                    - key: result
+                      value:
+                        stringValue: schema_mismatch
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asInt: "15"
+                  attributes:
+                    - key: result
+                      value:
+                        stringValue: success
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
               isMonotonic: true
             unit: '{requests}'
           - description: The number of values transmitted by the Directory Replication Agent per second.
@@ -238,8 +238,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: distingushed_names
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
                 - asDouble: 1
                   attributes:
                     - key: direction
@@ -248,8 +248,8 @@ resourceMetrics:
                     - key: type
                       value:
                         stringValue: other
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{values}/s'
           - description: The number of security descriptor propagation events that are queued for processing.
             name: active_directory.ds.security_descriptor_propagations_event.queued
@@ -257,28 +257,28 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "24"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{events}'
           - description: The rate of sub-operations performed.
             name: active_directory.ds.suboperation.rate
             sum:
               aggregationTemporality: 2
               dataPoints:
-                - asDouble: 26
-                  attributes:
-                    - key: type
-                      value:
-                        stringValue: security_descriptor_propagations_event
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
                 - asDouble: 25
                   attributes:
                     - key: type
                       value:
                         stringValue: search
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
+                - asDouble: 26
+                  attributes:
+                    - key: type
+                      value:
+                        stringValue: security_descriptor_propagations_event
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{suboperations}/s'
           - description: The number of threads in use by the directory service.
             name: active_directory.ds.thread.count
@@ -286,9 +286,9 @@ resourceMetrics:
               aggregationTemporality: 2
               dataPoints:
                 - asInt: "27"
-                  startTimeUnixNano: "1650474513913172400"
-                  timeUnixNano: "1650474513913172400"
+                  startTimeUnixNano: "1000000"
+                  timeUnixNano: "1000000"
             unit: '{threads}'
         scope:
-          name: otelcol/activedirectorydsreceiver
+          name: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver
           version: latest

